Quick start not working

### Description

I wanted to try out this project, and the quick start has some... rather obvious problems. Issues which don't really make this project seem trustworthy, if I'm being completely honest.

a) curl points to wrong URL for Caddyfile
b) Dockefile references Caddyfile in a `docker` subfolder, where the quickstart has both in the same folder (easily fixed, but still...)
c) API container failed to start due to DB schema errors

I've tried this both using a Windows docker host (WSL2) and a full Linux machine (I know from experience sometimes Windows vs Linux line-breaks in config files can cause strange issues) and had the same problems in both.

### Environment Information

- Deployment type: 
- Operating System: 
- Architecture: 
- Database: 
- Reverse Proxy: 
- Network setup: 


### Backend Version

main

### Web UI Version

N/A

### Package Format

Docker / OCI

### Expected Behavior

I'd expect the project to work when following the quick start guide...

### Observed Behavior

Obvious Dockerfile issues, later container failed to start (failed health check).

### Steps to Reproduce

Just follow the current quick-start guide...

### Logs
Attempting to run the quick-start commands:

```shell
curl: (22) The requested URL returned error: 404
```

After downloading the Caddyfile manually:

```shell
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/run/desktop/mnt/host/wsl/docker-desktop-bind-mounts/Ubuntu/a0ff709e2ea333663a277d6285b3ae7284490097997ce771e6fa65e28d5bf090" to rootfs at "/etc/caddy/Caddyfile": mount src=/run/desktop/mnt/host/wsl/docker-desktop-bind-mounts/Ubuntu/a0ff709e2ea333663a277d6285b3ae7284490097997ce771e6fa65e28d5bf090, dst=/etc/caddy/Caddyfile, dstFd=/proc/thread-self/fd/11, flags=MS_BIND|MS_REC: not a directory: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
```

After fixing Dockerfile:
`Error dependency dependency-track-apiserver failed to start`

```shell
2026-03-05 09:33:37,635 ERROR [UpgradeInitializer] An error occurred determining database schema version. Unable to continue.

alpine.server.upgrade.UpgradeException: An error occurred connecting to the database.

	at alpine.server.upgrade.UpgradeMetaProcessor.createConnection(UpgradeMetaProcessor.java:193)

	at alpine.server.upgrade.UpgradeMetaProcessor.<init>(UpgradeMetaProcessor.java:63)

	at org.dependencytrack.upgrade.UpgradeInitializer.contextInitialized(UpgradeInitializer.java:56)

	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:1049)

	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:624)

	at org.eclipse.jetty.server.handler.ContextHandler.contextInitialized(ContextHandler.java:984)

	at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:740)

	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:392)

	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1304)

	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:901)

	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:306)

	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:532)

	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93)

	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:171)

	at org.eclipse.jetty.server.Server.start(Server.java:470)

	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)

	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:89)

	at org.eclipse.jetty.server.Server.doStart(Server.java:415)

	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:93)

	at alpine.embedded.EmbeddedJettyServer.main(EmbeddedJettyServer.java:100)

Caused by: org.postgresql.util.PSQLException: FATAL: database "dependency_track" does not exist

	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2725)

	at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2837)

	at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:175)

	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:317)

	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)

	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:273)

	at org.postgresql.Driver.makeConnection(Driver.java:446)

	at org.postgresql.Driver.connect(Driver.java:298)

	at java.sql/java.sql.DriverManager.getConnection(Unknown Source)

	at java.sql/java.sql.DriverManager.getConnection(Unknown Source)

	at alpine.server.upgrade.UpgradeMetaProcessor.createConnection(UpgradeMetaProcessor.java:187)

	... 19 common frames omitted
```

### Additional Context

_No response_

### Pre-submission Checklist

- [x] I have searched existing issues to ensure this isn't a duplicate
- [x] I have redacted any sensitive information (passwords, tokens, private URLs)
- [x] I am using a supported version (not end-of-life)

--- a/README.md
+++ b/README.md
@@ -241,31 +241,7 @@ Extend Artifact Keeper with custom format handlers compiled to WebAssembly.
 
 ## Quick Start
 
-```bash
-mkdir artifact-keeper && cd artifact-keeper
-curl -fsSLO https://raw.githubusercontent.com/artifact-keeper/artifact-keeper/main/docker-compose.yml
-mkdir docker
-curl -fsSLo docker/Caddyfile https://raw.githubusercontent.com/artifact-keeper/artifact-keeper/main/docker/Caddyfile
-curl -fsSLo docker/init-db.sql https://raw.githubusercontent.com/artifact-keeper/artifact-keeper/main/docker/init-db.sql
-curl -fsSLo docker/init-pg-ssl.sh https://raw.githubusercontent.com/artifact-keeper/artifact-keeper/main/docker/init-pg-ssl.sh
-curl -fsSLo docker/init-dtrack.sh https://raw.githubusercontent.com/artifact-keeper/artifact-keeper/main/docker/init-dtrack.sh
-docker compose up -d
-```
-
-Open [http://localhost:30080](http://localhost:30080) and follow the first-time setup instructions.
-
-### Accessing from another machine on your network
-
-In development mode (`ENVIRONMENT=development`, the default), CORS automatically allows requests from private-network IPs (`192.168.x.x`, `10.x.x.x`, `172.16-31.x.x`) and localhost. No extra configuration is needed — just open `http://<server-ip>:30080` from any machine on your LAN.
-
-For production deployments, set `CORS_ORIGINS` in your `.env` to the public URL(s) users will access:
-
-```bash
-ENVIRONMENT=production
-CORS_ORIGINS=https://registry.example.com
-```
-
-**[Full Quickstart Guide →](https://artifactkeeper.com/docs/getting-started/quickstart/)**
+Get running in 5 minutes with Docker Compose: **[Quickstart Guide](https://artifactkeeper.com/docs/getting-started/quickstart/)**
 
 ## Documentation
 

